root/release/edit/tracklist.tt

```diff
@@ -66,6 +66,22 @@
         </label>
       </td>
     </tr>
+    <tr>
+    <td>
+      <label>
+      <input type="checkbox" name="use-custom-delimiter" data-bind="checked: useCustomDelimiter" />
+        [%= l('Use custom artist delimeter') %]
+      </label>
+    </td>
+      <td>
+        <label data-bind="visible: useCustomDelimiter">
+          [%= l('Custom delimiter') %]
```

Comment:
`add_colon(l('Custom delimiter'))` to separate the text from the input with some punctuation would be nice.